[BUG] Fails to build: TaskTimer.vala:35.42-35.50: error: value is less accessible than constant `GOFI.TaskTimer.UPDATE_INTERVAL'

Forwarded bug report from https://bugs.debian.org/1008351 ; not verified yet.

### Describe the bug
```
During a rebuild of all packages in sid, your package failed to build
on amd64.


Relevant part (hopefully):
> make[3]: Entering directory '/<<PKGBUILDDIR>>/obj-x86_64-linux-gnu'
> /<<PKGBUILDDIR>>/src/Services/TaskTimer.vala:35.42-35.50: error: value is less accessible than constant `GOFI.TaskTimer.UPDATE_INTERVAL'
>    35 |     public const int64 UPDATE_INTERVAL = 60 * US_C;
>       |                                          ^~~~~~~~~ 
> make[3]: *** [src/CMakeFiles/com.github.jmoerman.go-for-it-0.dir/build.make:299: src/com.github.jmoerman.go-for-it-0_valac.stamp] Error 1


The full build log is available from:
http://qa-logs.debian.net/2022/03/26/go-for-it_1.9.6-1_unstable.log
```

### Operating system
(assume Debian Sid/Unstable)

--- a/src/Services/TaskTimer.vala
+++ b/src/Services/TaskTimer.vala
@@ -32,7 +32,7 @@ public class GOFI.TaskTimer {
 
     private const int64 US_C = 1000000; // μs<->s conversion
 
-    public const int64 UPDATE_INTERVAL = 60 * US_C;
+    private const int64 UPDATE_INTERVAL = 60 * US_C;
 
     /**
      * A proxy attribute, that does not store any data itself, but provides

